db/structure.sql

```diff
@@ -2,12 +2,16 @@
 -- PostgreSQL database dump
 --
 
+-- Dumped from database version 9.5.0
```

Comment:
seems you've run the migration on a different PG version to the server - we'll need to do this properly (most likely via a the docker-compose pg server as we control that version)